src/BoundaryConditions/BoundaryConditions.jl

```diff
@@ -11,14 +11,16 @@ export
     apply_x_bcs!, apply_y_bcs!, apply_z_bcs!,
     fill_halo_regions!
 
-using CUDA
+using CUDA, Adapt
 using KernelAbstractions: @index, @kernel
 
 using Oceananigans.Architectures: CPU, GPU, device
 using Oceananigans.Utils: work_layout, launch!
 using Oceananigans.Operators: Ax, Ay, Az, volume
 using Oceananigans.Grids
 
+import Adapt: adapt_structure
```

Comment:
I'm not sure how the adapt_structures in boundary_condition.jl were working before but the method definition in boundary_condition_classifcation.jl fails if this isn't here

Reply:
Probably it was not needed before